vector/src/main/java/im/vector/app/features/call/CallAudioManager.kt

```diff
@@ -150,6 +154,7 @@ class CallAudioManager(
 
     fun onCallConnected(mxCall: MxCall) {
         Timber.v("##VOIP: AudioManager call answered, adjusting current sound device")
+        setupAudioManager(mxCall)
         adjustCurrentSoundDevice(mxCall)
```

Comment:
I think there is no need to call that anymore as it is called at the end of #setupAudioManager()

Reply:
Agreed! Changed in aa2a24e